server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileLookup.java

```diff
@@ -71,27 +72,18 @@ private void incrementAncestors(QProfile profile, List<QProfile> ancestors, DbSe
       if (parentDto == null) {
         throw new IllegalStateException("Cannot find parent of profile : " + profile.id());
       }
-      QProfile parent = QProfile.from(parentDto);
+      QProfile parent = QProfile.from(parentDto, null);
       ancestors.add(parent);
       incrementAncestors(parent, ancestors, session);
     }
   }
 
-  private static List<QProfile> toQProfiles(List<QualityProfileDto> dtos) {
-    return newArrayList(Iterables.transform(dtos, ToQProfile.INSTANCE));
+  private static List<QProfile> toQProfiles(List<QualityProfileDto> dtos, OrganizationDto organization) {
+    return dtos.stream().map(dto -> QProfile.from(dto, organization)).collect(toList(dtos.size()));
   }
 
   @CheckForNull
   private QualityProfileDto findQualityProfile(String name, String language, DbSession session) {
     return db.qualityProfileDao().selectByNameAndLanguage(name, language, session);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "selectByNameAndLanguage"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)